client/my-sites/checkout/composite-checkout/record-analytics.js

```diff
@@ -254,12 +254,10 @@ export default function createAnalyticsEventHandler( reduxDispatch ) {
 					);
 				}
 				case 'THANK_YOU_URL_GENERATED':
-					return logStashEvent( 'thank you url generated', {
-						url: action.payload.url,
-					} );
-				case 'EMPTY_CART_CTA_CLICKED':
 					return reduxDispatch(
-						recordTracksEvent( 'calypso_checkout_composite_empty_cart_clicked' )
+						logStashEventAction( 'thank you url generated', {
```

Comment:
Looks like the rebase might have introduced a bug here, probably due to a recent change in how these events are recorded. `logStashEventAction` no longer exists, so this should be reverted to `logStashEvent` without redux.

Reply:
This should be resolved in the latest commit

https://github.com/Automattic/wp-calypso/blob/c42834476a9ef02a215c0e235c15bc6f1f2246e7/client/my-sites/checkout/composite-checkout/record-analytics.js#L256-L259